api/client/service/opts.go

```diff
@@ -528,6 +531,7 @@ const (
 	flagEnv                  = "env"
 	flagEnvRemove            = "env-rm"
 	flagEnvAdd               = "env-add"
+	flagGroupAdd             = "group-add"
```

Comment:
Thanks @cpuguy83. The pull request has been updated and now `docker service update` support `--group-rm`. Please let me know if there are any issues.